app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user, foreign_key: 'user_id', class_name: 'Post'
+  belongs_to :post
+
```

Comment:
- [ ] As pointed out by the previous reviewer, please ensure that this belongs to the `author` instead of `user`